.github/pull_request_template.md

```diff
@@ -5,20 +5,29 @@
 3. Make sure documentation is updated for your PR!
 
 -->
+# Description
```

Comment:
@leonlnj thanks for the quick review! I made one more small change to the PR template. Please take a look and lmk if you have any comments for change.